errai-codegen/src/main/java/org/jboss/errai/codegen/meta/RuntimeAnnotation.java

```diff
@@ -16,25 +16,32 @@
 
 package org.jboss.errai.codegen.meta;
 
+import javax.enterprise.util.Nonbinding;
 import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
 import java.util.Arrays;
+import java.util.Map;
+
+import static java.util.stream.Collectors.toMap;
 
 /**
  * @author Tiago Bento <tfernand@redhat.com>
  */
-public class RuntimeMetaAnnotation extends MetaAnnotation {
+public class RuntimeAnnotation extends MetaAnnotation {
```

Comment:
Do you think we could change this to match the naming convention of the other MetaClass impls in this module and call this "JavaReflectionAnnotation"?

Reply:
Sure :)